### PR TITLE
New Hakka Cheat Sheet

### PR DESCRIPTION
**Pull request by hliu45**
## Description of new Instant Answer, or changes

It provides a quick solution to checkout Hakka language. 
IA: https://duck.co/ia/view/hakka_cheat_sheet

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/hakka_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by hliu45** [comment]
Thank you. I added the link to IA page at the buttom.

**Comment by sahildua2305** [comment]
@duckduckgo/duckduckhack-contributors need your feedback on this.

File: share/goodie/cheat_sheets/json/hakka.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    "aliases": [
+        "kejia"
+    ],
+
+    "template_type": "language",
+
+    "section_order": [
+        "Basic",
+        "Greeting",
+	"Etiquette",
```
Comment (kirkins) [inline share/goodie/cheat_sheets/json/hakka.json]: Can you replace any lines where you're using tabs like this one with 4 spaces.

Looks like all the lines with "trn" use tabs too.

**Comment by mintsoft** [comment]
Ping! @hliu45 Are you OK to finish this off? If not, it's fine we can open it up to the community

**Comment by hliu45** [comment]
Hi Rob,

I am currently working on another project, and don't follow up the
community recently. My apology. I am still can finish this off, but if you
want to open it up to the community. It's ok.


Thank you for checking.

Best,

Joe Liu

**Comment by hliu45** [comment]
Hi Rob,

Can I get back to you next week? Sorry that I have been busy since two
months ago. Very sorry.


Joe

**Comment by mintsoft** [comment]
@hliu45 that's fine, I'll leave it open for now. :+1:

**Comment by mintsoft** [comment]
@hliu45 I'm going to close this for now as I'm doing some housekeeping. Feel free to re-open it when you get the chance to continue with this 👍

**Pull request closed without merging**